config/unix-cuda.cmake

```diff
@@ -18,23 +22,24 @@ include_guard(GLOBAL)
 # - https://devblogs.nvidia.com/building-cuda-applications-cmake/
 
 # Discover CUDA compute capabilities.
-if( NOT DEFINED Draco_CUDA_ARCH )
+if( NOT DEFINED CUDA_ARCHITECTURES )
```

Comment:
Is the `query_gpu` path still here for older versions of CMake?

Reply:
As far as I can tell, cmake still doesn't actually do that introspection.  Please let me know if I am wrong about this.

Reply:
OK, got it, I see that after looking at more of the documentation.

It looks like `CMAKE_CUDA_ARCHITECTURES` is supposed to be initialized with a default value but it doesn't seem to be (I tried printing it during configure) so I think it makes sense to leave in the logic to print the value for now.